addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3108,7 +3107,9 @@ export class OdooEditor extends EventTarget {
      * @param {Node} node
      */
     _cleanForPaste(node) {
-        if (!this._isWhitelisted(node) || this._isBlacklisted(node)) {
+        if (!this._isWhitelisted(node) || this._isBlacklisted(node) ||
+            node.id && node.id.startsWith('docs-internal-guid') ||
+            node.nodeName === 'P' && node.parentNode.nodeName === 'LI') {
```

Comment:
Isn't that handled by `sanitize` already ? I think it's precisely the sanitization case that we saw on Friday with @Zinston.

Reply:
@dmo-odoo I think that is for removing empty block in li.

Reply:
Mmmh... yes you're right. Does it make sense with what we saw on Friday @Zinston ? Why only remove them when the P is empty ? Shouldn't we remove them all the time ?

Reply:
@visp-odoo We're not sure whether we want to always remove P in LI or never remove them, but in both cases, we'll do it in sanitize. Now we wonder, did you see an issue with P inside LI that made you try to remove them ?

Reply:
@dmo-odoo I show that when we copy paste list from gdocs then it adds unwanted margin to list compare to editor list and as per example given in the spec it creates problem. So, thats the reason i try to remove that `p`.

Reply:
@visp-odoo I see. Could you try to adapt the condition in sanitize so that it always unwrap P inside LI, even when it is not empty ?

Reply:
ok, i will try it.

Reply:
@dmo-odoo Done changes.